#! /bin/sh /usr/share/dpatch/dpatch-run
## 90-patch-mpc-basedir.dpatch by  <pgquiles@elpauer.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Set MPC base dir directly, no need to fix the tests performed by ACE 
## DP: because there is no Debian/VMS (yeat :-)

@DPATCH@

diff -rupd ACE_wrappers.orig/bin/mpc.pl ACE_wrappers/bin/mpc.pl
--- ACE_wrappers.orig/bin/mpc.pl	2009-11-23 01:19:57.000000000 +0100
+++ ACE_wrappers/bin/mpc.pl	2009-11-24 00:24:27.000000000 +0100
@@ -20,13 +20,7 @@ use FindBin;
 use File::Spec;
 use File::Basename;
 
-my($basePath) = $FindBin::RealBin;
-my($baseName) = $FindBin::RealScript;
-if ($^O eq 'VMS') {
-  $basePath = File::Spec->rel2abs(dirname($0)) if ($basePath eq '');
-  $basePath = VMS::Filespec::unixify($basePath);
-}
-$basePath .= '/MakeProjectCreator';
+my($basePath) = '/usr/share/ace/bin/MakeProjectCreator';
 
 my($mpcroot) = $ENV{MPC_ROOT};
 my($mpcpath) = (defined $mpcroot ? $mpcroot :
diff -rupd ACE_wrappers.orig/bin/mwc.pl ACE_wrappers/bin/mwc.pl
--- ACE_wrappers.orig/bin/mwc.pl	2009-11-23 01:19:57.000000000 +0100
+++ ACE_wrappers/bin/mwc.pl	2009-11-24 00:23:44.000000000 +0100
@@ -20,13 +20,7 @@ use FindBin;
 use File::Spec;
 use File::Basename;
 
-my($basePath) = $FindBin::RealBin;
-my($baseName) = $FindBin::RealScript;
-if ($^O eq 'VMS') {
-  $basePath = File::Spec->rel2abs(dirname($0)) if ($basePath eq '');
-  $basePath = VMS::Filespec::unixify($basePath);
-}
-$basePath .= '/MakeProjectCreator';
+my($basePath) = '/usr/share/ace/bin/MakeProjectCreator';
 
 my($mpcroot) = $ENV{MPC_ROOT};
 my($mpcpath) = (defined $mpcroot ? $mpcroot :
